packages/testsuite/cypress.config.ts

```diff
@@ -30,50 +34,156 @@ export default defineConfig({
                 },
               ])
               .withWaitStrategy(Wait.forLogMessage(new RegExp(".*(WildFly Full.*|JBoss EAP.*)started in.*")))
-              .withStartupTimeout(333000)
-              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[])
+              .withStartupTimeout(333000);
+            if (useNetworkHostMode === true) {
+              console.log("host mode");
+              findAPortNotInUse(8080, 8180)
+                .then((freePort) => {
+                  portOffset = freePort - 8080;
+                  wildfly
+                    .withNetworkMode("host")
+                    .withCommand([
+                      "-c",
+                      configuration || "standalone-insecure.xml",
+                      `-Djboss.socket.binding.port-offset=${portOffset.toString()}`,
+                    ] as string[]);
+                })
+                .catch((error) => {
+                  console.log(error);
+                });
+            } else {
+              console.log(`network mode, network name: ${config.env.NETWORK_NAME as string}`);
+              wildfly
+                .withNetworkMode(config.env.NETWORK_NAME as string)
+                .withNetworkAliases("wildfly")
+                .withExposedPorts(9990)
+                .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[]);
+            }
+            wildfly
               .start()
               .then((wildflyContainer) => {
+                const managementPortWithOffset = portOffset + 9990;
                 startedContainers.set(name as string, wildflyContainer);
-                const managementApi = `http://localhost:${wildflyContainer.getMappedPort(9990)}/management`;
-                return axios
-                  .post(managementApi, {
-                    operation: "list-add",
-                    address: ["core-service", "management", "management-interface", "http-interface"],
-                    name: "allowed-origins",
-                    value: `http://localhost:${config.env.HAL_CONTAINER_PORT as string}`,
-                  })
-                  .then(() => {
-                    return axios.post(managementApi, {
-                      operation: "reload",
-                    });
-                  })
-                  .then(() => {
-                    const startTime = new Date().getTime();
-                    const interval = setInterval(() => {
-                      if (new Date().getTime() - startTime > 10000) {
-                        clearInterval(interval);
-                        reject();
-                      }
-                      axios
-                        .post(managementApi, {
-                          operation: "read-attribute",
-                          name: "server-state",
-                        })
+                if (useNetworkHostMode === true) {
+                  startedContainersManagementPorts.set(name as string, portOffset + 9990);
```

Comment:
No here we need it because the we need the management port is used for connection to the container in method `execute:in:container` [see line](https://github.com/hal/berg/pull/77/files#diff-cb94efd61fcb80a72acd2fab4aa1810affd6d7ad7f7431b6e9ea420077f06bdfR292)

Reply:
Of course we need it here. But where is the clean-up of the value that is set here?

Reply:
clean-up added.